bin/install.sh

```diff
@@ -111,12 +111,14 @@ fishFunctionsDir="${PREFIX}/fish_functions"
 mkdir -p "${fishFunctionsDir}"
 cat <<'EOF' >"${fishFunctionsDir}/okta-aws.fish"
 function okta-aws
-    withokta "aws --profile $argv[1]" $argv
+    set -lx OKTA_PROFILE "$argv[1]"
+    OKTA_PROFILE="$1" withokta "aws --profile $argv[1]" $argv[2..-1]
```

Comment:
Same as below.  Doesn't work the same in fish as bash.

Reply:
fixed